OMEdit/OMEditLIB/Annotations/BitmapAnnotation.cpp

```diff
@@ -168,6 +168,7 @@ void BitmapAnnotation::parseShapeAnnotation()
   GraphicItem::parseShapeAnnotation(mpBitmap);
 
   mExtent = mpBitmap->getExtent();
+  mExtent.evaluate(mpBitmap->getParentModel());
```

Comment:
`evaluate` can throw an exception if the expression is invalid in some way, so you probably want to catch those somehow.

Reply:
Isn't it the `fromExp` that will throw the exception?

Reply:
With the latest commit it looks good. I also noticed that I pointed out the wrong location since I read this as calling `Expression::evaluate` at first, but now I see that it's only in `DynamicAnnotation` you do that.